solr/core/src/java/org/apache/solr/cloud/CreateAliasCmd.java

```diff
@@ -68,34 +249,100 @@ public void call(ClusterState state, ZkNodeProps message, NamedList results)
     Thread.sleep(100);
   }
 
+  private Map<String, String> buildAliasMap(String routedField, String routingType, String tz, String increment, String maxFutureMs, ZkNodeProps collectionProps) {
+    Map<String, Object> properties = collectionProps.getProperties();
+    Map<String,String> cleanMap = properties.entrySet().stream()
+        .filter(stringObjectEntry ->
+            !"fromApi".equals(stringObjectEntry.getKey())
+                && !"stateFormat".equals(stringObjectEntry.getKey())
+                && !"name".equals(stringObjectEntry.getKey()))
+        .collect(Collectors.toMap((e) -> "collection-create." + e.getKey(), e -> String.valueOf(e.getValue())));
+    cleanMap.put(ROUTING_FIELD, routedField);
+    cleanMap.put(ROUTING_TYPE, routingType);
+    cleanMap.put(ROUTING_INCREMENT, increment);
+    cleanMap.put(ROUTING_MAX_FUTURE, maxFutureMs);
+    cleanMap.put(TZ, tz);
+    return cleanMap;
+  }
+
+  private Instant validateStart(TimeZone zone, DateTimeFormatter fmt, String start) {
```

Comment:
This appears redundant with using DateMathParser earlier and I'm sure we don't need so much code for this matter (ignore millisecond truncation; in practice the user is going to do something like `/MONTH` or `/DAY` for start.

Reply:
checking the ms was the main point, the format and reparse was meant to trap any timezone/formatter craziness, which mostly goes away with the collection creation. Simplified, in update